src/views/Task/Create.js

```diff
@@ -1,29 +1,99 @@
-import React from 'react'
+import React, { Component } from 'react'
 import Layout from '../../components/Layout'
+import task from '../../api/task'
+import DatePicker from '../../components/DatePicker'
+import moment from 'moment'
 
-export default ({ children }) => (
-  <Layout>
-    <h2>Create Task</h2>
-    <label>
-      Title
-      <input type="text" />
-    </label>
-    <label>
-      Starting Date
-      <input type="date" />
-    </label>
-    <label>
-      Deadline
-      <input type="date" />
-    </label>
-    <label>
-      Description
-      <textarea></textarea>
-    </label>
-    <label>
-      Urgency
-      <input type="text" />
-    </label>
-    <input type="submit" value="Create Task" />
-  </Layout>
-)
+export default class Create extends Component {
+
+  state = {
+    startDate: moment(),
+    endDate: moment(),
+    title: "",
+    description: "",
+    urgency: ""
+  }
+
+
+  /**
+   * Creates task based on data in inputfields
+   */
+  createTaskOnClick = () => {
+    const taskData = {
+      title: this.state.title,
+      description: this.state.description,
+      urgency: this.state.urgency,
+      startDate: this.state.startDate,
+      endDate: this.state.endDate
+    }
```

Comment:
No need to create the object again, we could just pass the state directly to the `task.create` method.
```js
task.create(this.state)
```